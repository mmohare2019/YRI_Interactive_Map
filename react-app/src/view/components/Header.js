import React from "react";
import "../screen/Home.css"
import { Link } from "react-router-dom";

function Header() {
    return (
        <nav class="header">
            <img class="yri-logo" src="YRI_LOGO.png" alt="YRI logo"/>
            <img class="ccsj-logo" src="ccsjLogo.jpg" alt="CCSJ logo"/> 
            <Link to="/login">Admin Login</Link> 
        </nav>
    );
}

export default Header