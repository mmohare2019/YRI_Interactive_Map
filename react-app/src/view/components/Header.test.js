import TestRenderer from "react-test-renderer";
import Header from "./Header";
import React from "react";
import renderer from 'react-test-renderer';

describe("Header render tests", () => {
    test("render smoke test", () => {
        const testRenderer = TestRenderer.create(<Header/>)
    })
    
});

test("header test", function () {
    const component = renderer.create(<Header/>).toJSON();
    expect(component).toMatchSnapshot(); 
});