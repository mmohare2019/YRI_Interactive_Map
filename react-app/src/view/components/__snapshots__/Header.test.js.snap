// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header test 1`] = `
<nav
  className="header navbar navbar-expand navbar-light"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/admin"
    >
      <img
        alt="YRI logo"
        className="yri-logo"
        height="30"
        src="YRI_LOGO.png"
        width="30"
      />
      <img
        alt="CCSJ logo"
        className="ccsj-logo"
        height="30"
        src="ccsjLogo.jpg"
        width="30"
      />
      <button
        aria-controls="navbarNavAltMarkup"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-target="#navbarNavAltMarkup"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
    </a>
    <div
      className="collapse navbar-collapse"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav"
      >
        <a
          className="nav-item nav-link"
          href="/"
        >
          Home
        </a>
        <a
          className="nav-item nav-link"
          href="/key"
        >
          Category key
        </a>
        <a
          className="nav-item nav-link"
          href="/login"
        >
          Admin login
        </a>
      </div>
    </div>
  </div>
</nav>
`;
